src/cinder/Signals.cpp

```diff
@@ -111,6 +111,22 @@ ScopedConnection& ScopedConnection::operator=( ScopedConnection &&rhs )
 	return *this;
 }
 
+ConnectionList::~ConnectionList()
+{
+	clear();
+}
+
+void ConnectionList::add( Connection &&target )
+{
+	mConnections.emplace_back( target );
+}
+
+void ConnectionList::clear()
+{
+	for( auto &conn : mConnections )
```

Comment:
So talking with @andrewfb here, turns out that `std::weak_ptr` doesn't have a move constructor or assignment operator until C++14, and vc120 must not support this yet. So I think the interim solution is to leave that `.reset()` in there and mark it with a TODO. The rest of the operator has no effect as `rhs` is about to die, so I'll remove those.